service_mesh/v2x/ossm-troubleshooting-istio.adoc

```diff
@@ -0,0 +1,68 @@
+[id="ossm-troubleshooting"]
+= Troubleshooting your service mesh
+include::modules/ossm-document-attributes.adoc[]
+:context: troubleshooting-ossm
+
+toc::[]
+
+This section describes how to identify and resolve common problems in {ProductName}. Use the following sections to help troubleshoot and debug problems when deploying {ProductName} on {product-title}.
+
+// The following include statements pull in the module files that comprise the assembly.
+
+include::modules/ossm-understanding-versions.adoc[leveloffset=+1]
+
+== Troubleshooting Operator installation
+
+In addition to the information in this section, be sure to review the following topics:
+
+* xref:../../operators/understanding/olm-what-operators-are.adoc[What are Operators?].
+
+* xref:../../operators/understanding/olm/olm-understanding-olm.adoc[Operator Lifecycle Management concepts].
+
+* xref:../../support/troubleshooting/troubleshooting-operator-issues.adoc[OpenShift Operator troubleshooting section].
+
+* xref:../../support/troubleshooting/troubleshooting-installations.adoc[OpenShift installation troubleshooting section]
```

Comment:
Either remove all `.` or add one for this last point? Doesn't seem like they should be required for links